samples/nhibernate/simple/NHibernate_7/Client/App.config

```diff
@@ -0,0 +1,20 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <configSections>
+    <section name="UnicastBusConfig"
+             type="NServiceBus.Config.UnicastBusConfig, NServiceBus.Core"/>
+  </configSections>
+
+  <connectionStrings>
+    <add name="NServiceBus/Persistence"
+         connectionString="Data Source=.;Database=Samples.NHibernate;Integrated Security=True"
+         providerName="System.Data.SqlClient" />
+  </connectionStrings>
+  
+  <UnicastBusConfig>
+    <MessageEndpointMappings>
```

Comment:
@SzymonPobiega we could go either way, depending what's the goal at hand:

-If we want to show case how things are done in v7, is this the right sample for it?
-Should we keep the v6 vs v7 samples as close as possible?